test/classes/initializers/generics/varargs-type.future

```diff
@@ -0,0 +1,3 @@
+bug: Unresolved call for initializer that uses type of varargs formal
+
+Based on 'test/functions/nspark/generic-varargs2.chpl'
```

Comment:
Please explain in this .future how this test differs from nspark's and/or what testing value this test adds.

Reply:
Hmm, not sure what I was thinking... I'll remove the new test.